yarn.lock

```diff
@@ -0,0 +1,4868 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Try removing the `yarn.lock` and using npm – not sure why the snapshots are failing, but this repo using `npm` at the moment